macos/FluentUI/Button/Button.swift

```diff
@@ -431,11 +473,19 @@ class ButtonCell: NSButtonCell {
 			break
 		}
 
+		// First, center the Primary Image
 		var x = (rect.width - imageSize.width) / 2
 		var y = (rect.height - imageSize.height) / 2
 
 		if xOffsetSign != 0 {
+			// Second, offset the Primary Image from the Title
 			x += CGFloat(xOffsetSign) * (titleSize.width + titleToImageSpacing) / 2
+
+			// Third, offset the Title from the Secondary Image
+			if let controlView = self.controlView as? Button,
```

Comment:
minor: can omit self. here - I think type inference easily gets the `if let property = someProperty as? SomeType` pattern